manifests/init.pp

```diff
@@ -180,12 +192,21 @@
 ) inherits katello::params {
 
   include katello::repo
-  include katello::candlepin
-  include katello::qpid
-  include katello::pulp
-  Class['katello::repo'] -> Class['katello::pulp']
-  include katello::application
-  Class['katello::repo'] -> Class['katello::application']
-  Class['katello::qpid'] -> Class['katello::candlepin'] -> Class['katello::application']
-
+  if $manage_candlepin {
+    include katello::candlepin
+  }
+  if $manage_qpid {
+    include katello::qpid
+  }
+  if $manage_pulp {
+    include katello::pulp
+    Class['katello::repo'] -> Class['katello::pulp']
+  }
+  if $manage_application {
+    include katello::application
+    Class['katello::repo'] -> Class['katello::application']
+    if $manage_qpid and $manage_candlepin {
+      Class['katello::qpid'] -> Class['katello::candlepin'] -> Class['katello::application']
```

Comment:
I think this should be split: `Class['katello::qpid'] -> Class['katello::candlepin']` should be under `$manage_candlepin` as a separate `if $manage_qpid`. Then this if block should just be `if $manage_candlepin`

Reply:
Thanks - I simplified the ordering of classes in the following commit.